website/pages/docs/deployment/ecs.md

```diff
@@ -0,0 +1,379 @@
+
+In this tutorial we will be deploying CloudQuery on AWS ECS using Fargate. You will be using the AWS CLI to create the required resources. You can also use the AWS Console to create the resources. At the end of the tutorial you will have a CloudQuery instance running on AWS ECS that will collect data from your AWS account and store it in an S3 bucket. You can then query the data using Athena.
+
+## Prerequisites
+Before starting the deployment process, you need to have the following prerequisites:
+  * An AWS account
+  * AWS CLI installed and configured
+  * Basic understanding of AWS ECS and Fargate
+
+## Step 1: Generate a CloudQuery Configuration File
+
+Create a new file named `cloudquery.yml` with the following contents:
+```yaml
+kind: source
+spec:
+  # Source spec section
+  name: aws
+  path: "cloudquery/aws"
+  version: "v15.3.0"
+  tables: ["*"]
+  destinations: ["s3"] 
+---
+kind: destination
+spec:
+  name: "s3"
+  path: "cloudquery/s3"
+  version: "v2.2.3"
+  write_mode: "append"
+  spec:
+    bucket: <REPLACE_WITH_S3_DESTINATION_BUCKET>
+    path: "{{TABLE}}/{{UUID}}.parquet"
+    format: "parquet"
+    athena: true
+```
+
+This will create a configuration file that will instruct CloudQuery to collect data from AWS and store it in an S3 bucket. You will need to replace the `REPLACE_WITH_S3_DESTINATION_BUCKET` placeholder with the name of the S3 bucket you want to use to store the data. You can also modify the configuration file to collect only the data you need. For more information on how to create a configuration file, [visit our docs](/docs/reference/source-spec)
+
+
+In order to inject the config file into the prebuilt container you will have to base64 encode the contents of the `cloudquery.yml` file . Assuming you are running on a Linux or MacOS machine you can do this conversion by running the following command:
+```bash
+cat cloudquery.yml | base64
+```
+
+## Step 2: Create an ECS Cluster
+The first step in deploying CloudQuery on AWS ECS is to create an ECS cluster. To create an ECS cluster, use the following command:
+
+Prior to running replace `<REPLACE_ECS_CLUSTER_NAME>` with the name you want to give to your ECS cluster.
+```bash
+aws ecs create-cluster --cluster-name <REPLACE_ECS_CLUSTER_NAME>
+```
+
+
+## Step 3: Create a Log Group
+The next step is to create a log group for your ECS task. To create a log group, use the following command:
+
+Prior to running replace `<REPLACE_LOG_GROUP_NAME>` with the name you want to give to your log group.
+```bash
+aws logs create-log-group --log-group-name <REPLACE_LOG_GROUP_NAME>
+```
+
+## Step 4: Set Log Group Retention
+After creating a log group, you need to set the retention policy for your log group. To set the retention policy, use the following command:
+
+Replace `<REPLACE_LOG_GROUP_NAME>` with the name of the log group that you created in Step 3.
+
+```bash
+aws logs put-retention-policy --log-group-name <REPLACE_LOG_GROUP_NAME> --retention-in-days 14
```

Comment:
```suggestion
Replace `<REPLACE_WITH_LOG_GROUP_NAME>` with the name of the log group that you created in Step 3.

```bash
aws logs put-retention-policy --log-group-name <REPLACE_WITH_LOG_GROUP_NAME> --retention-in-days 14
```